www/src/page-layout.html

```diff
@@ -9,7 +9,7 @@
     <title>{{ config.mail_domain }} {{ pagename }}</title>
     <link rel="stylesheet" href="./main.css">
     <link rel="icon" href="/logo.svg">
-    <link rel=”mask-icon” href=”/logo.svg” color=”#000000">
+    <link rel="mask-icon" href="/logo.svg" color="#000000">
```

Comment:
I understand why you want to change this, but please revert this. This might cause merge conflicts for other operators when they want to upgrade their instance; many of them don't know git well enough to deal with this. We should not cause a merge conflict so easily.